src/engine/bank06.asm

```diff
@@ -351,13 +389,13 @@ Func_006_4248:
 	bank1call OpenCardPage_FromCheckPlayArea
 	ret
 
```

Comment:
A few things up to this line:

- Correct me if I'm wrong, but I think that the functions currently named `ShowOpponentHand_HandlePlayAreaView`, `ShowPlayerHand_HandlePlayAreaView`,  `PrintCardName_HandlePlayAreaView` can all be made local to `HandlePlayAreaView` as the main function handler/loop. They appear to be branches or subroutines unique to `HandlePlayAreaView`.

- Doing the above would be particularly convenient for renaming `ShowOpponentHand_HandlePlayAreaView_2` to `ShowOpponentHand_HandlePlayAreaView`.

- But speaking of that, I would reword those to `HandlePlayAreaView_SomethingSpecific`. The global context as the prefix is the trend in other places of this repository, and personally I think it's better visually.

- Now, this one it's just semantics but, I think Play Area refers only to Arena + bench. To be honest, this is quite confusing because in the Check menu the menu items specifically say Play Area, which contradicts other in-game references of Play Area being just those two locations. In the repository, the play area screen (in bank 1 code) refers to the screen shown by pressing B+UP/DOWN from the main battle screen. This is also a subview of the check screen, which is triggered by pressing up in the top or down on the bottom of the check screen. The thing is the Check submenu that triggers HandlePlayAreaView is specifically called "In Play Area" so I'm not sure what would be the most appropriate way to handle this... Anyway, to clarify, I'm not mistaken, HandlePlayAreaView only covers the "In Play Area" check submenu, is that right? (that is, it doesn't overlap with the other 3 submenus: "Glossary" / "Your Play Area" / "Opp. Play Area"). If that's the case there's probably some wrapper (I think Func_3b31) that takes care of of that four item Check submenu that eventually may lead to your `HandlePlayAreaView`. So, since we don't have enough information yet, I guess we can leave it like this, but stay alert in case eventually they should be mass renamed to some syntax like "CheckMenu_InPlayArea" (along, with "CheckMenu_Glossary", etc)

- Anyway, to finish for now, I think constants for `wPlayAreaCursorPosition` would be really useful! Same issue with the naming as above, but for the reasons I explained above, we can forget about semantics for now. Maybe something like e.g. `PLAYAREAVIEW_ARENA`, `PLAYAREAVIEW_DISCARD_PILE`, etc?

Thanks and sorry for the wall of text :P

Reply:
Regarding point 4, I've just noticed that #53 does cover the Check menu, so it should clarify some things regarding it.

Reply:
1. Actually, I was think about it, and decided to make them global as they were directed from the table data. But with your comment, now I think it might be better to make them local.
3. I think so. And I prefer to name things in that way.
4. Maybe, to prevent the word from confusing us, we could use a new word, like 'Field' or 'Table' representing the playing board itself.

Reply:
I think, eventually, the most readable/cleanest way could be to converge to some syntax like `CheckMenu_InPlayArea_Something`, or just `InPlayArea_Something`, etc. Since #53 covers many things also related to this but kind of a bit more top level (Check menus, draw the In Play Area graphics, etc) for now I think I'm going to try to close #53 and then see how we could approach this.

Reply:
Yeah. I agree.

Reply:
Following the names of the bank 2 counterparts (``OpenYourOrOppPlayAreaScreen_*`) these should probably be ``OpenInPlayAreaScreen_*``. For example, ``OpenInPlayAreaScreen_TurnHolderHand`` and ``OpenInPlayAreaScreen_NonTurnHolderHand``. In practice Player/Opponent is probably always accurate but since the functions check the ``hWhoseTurn`` address I'd use "TurnHolder" and "NonTurnHolder", also for consistency with the bank 2 functions.